### ISSUE TITLE
Linux: Support Raspberry Pi OS (32 bit)

### ISSUE THREAD

**Issue by ScottMorris** [microsoft/vscode#108460]
When trying to run the Insiders version of VS Code on a fresh Raspberry Pi OS install it errors out with a _relocation error_, the whole error can be found below.

- VSCode Version: code-insiders_1.51.0-1602221704
- OS Version: Raspberry Pi OS August 2020
- Hardware: Raspberry Pi 3B+

Steps to Reproduce:

Setting up Raspberry Pi
1. Download [Raspberry Pi OS](https://www.raspberrypi.org/downloads/raspberry-pi-os/)
1. Flash it to an SD Card
1. Go through First Run wizard
    1. Connect to the Internet
    1. Install the latest updates
1. Reboot

Installing VS Code
1. [Download VSCode Insiders](https://code.visualstudio.com/insiders/): the ARM `.deb` package of VS Code.
1. Install with `sudo apt install ./Downloads/code-insiders_1.51.0-1602221704_armhf.deb`
1. Run it from the command line, `code-insiders`
1. Get error

```bash
pi@raspberrypi:~ $ code-insiders 
/usr/share/code-insiders/bin/../code-insiders: relocation error: /usr/share/code-insiders/bin/../code-insiders: symbol ��� version GLIBC_2.4 not defined in file libc.so.6 with link time reference
```

Does this issue occur when all extensions are disabled?: **Yes**

**Comment by ScottMorris** [comment]
The issue pointed out by _vscodebot_ is not related to this issue.

**Comment by joaomoreno** [comment]
Just looks like a libc incompatibility, unfortunately. Can't really fix it for now.

**Comment by joaomoreno** [comment]
Related: https://github.com/electron/electron/issues/16205

**Comment by joaomoreno** [comment]
I can't reproduce this. Just installed RPiOS fresh on my RPi4, ran updates, installed the deb and Code Insiders runs just fine.

It's possible that there's a difference between RPi3 and 4... but I doubt it. @ScottMorris do you mind trying it all over again?

**Comment by ScottMorris** [comment]
I will test this again this evening, to double check, which version of Raspbian did you run?  I installed the following one,

![image](https://user-images.githubusercontent.com/404967/98824611-83383c00-2401-11eb-84a9-374e1d6e097b.png)

There are three different issues of it and I didn't specify which vanilla one I used.  My bad.

![image](https://user-images.githubusercontent.com/404967/98824752-a82caf00-2401-11eb-922c-495bba74b8bc.png)

**Comment by ScottMorris** [comment]
I reflashed my boot device and refollowed my steps (including the updates) with a the most recent version of Code Insiders from the website. I used the `code-insiders_1.52.0-1605086481_armhf.deb` package and everything worked!

![2020-11-11-102314_1920x1080_scrot](https://user-images.githubusercontent.com/404967/98831873-1ffed780-240a-11eb-8c20-0bab7daad9dd.png)

Looking back at my original version, `code-insiders_1.51.0-1602221704` there has been a release or two between then and now.  Thanks so much for your efforts.  I'm looking forward to playing more with VS Code on the Raspberry Pi.
